metrics/metrics.go

```diff
@@ -0,0 +1,120 @@
+package metrics
+
+import (
+	"time"
+
+	viperutil "github.com/Conflux-Chain/go-conflux-util/viper"
+	"github.com/ethereum/go-ethereum/metrics"
```

Comment:
Strongly recommend to use `github.com/rcrowley/go-metrics` instead, and remove helper methods in the bottom.

Reply:
`github.com/rcrowley/go-metrics` seems not to be rather robust with influxdb integration.